src/doppkit_toolbox/esri/toolboxes/doppkit_wrapper_toolbox.pyt

```diff
@@ -116,25 +109,18 @@ class FetchExport:
     def execute(self, parameters, messages):
         """The source code of the tool."""
 
-        arcpy.AddMessage("====== PARAMETERS ======")
-        for parameter_print in parameters:
-            if parameter_print.displayName in [
-                "GRiD Access Token",
-                "Download Directory",
-                "Add Files to Map?",
-            ]:
-                pass
-            else:
-                arcpy.AddMessage(
-                    f"{parameter_print.displayName} = {parameter_print.valueAsText}"
-                )
-
         named_parameters = SyncParameters(*parameters)
 
         token = named_parameters.token.valueAsText
         url = named_parameters.grid_server.valueAsText
         log_level = "DEBUG"
 
+        arcpy.AddMessage("====== PARAMETERS ======")
+        for parameter_print in named_parameters:
+            arcpy.AddMessage(
+                f"{parameter_print.displayName} = {parameter_print.valueAsText}"
```

Comment:
for the future- this will print the GRiD Access token to the details pane in plain text. If we feel as though this is a security issue, this should be slightly modified.